CMakeLists.txt

```diff
@@ -5,7 +5,28 @@ project(minimum_distance_plugin)
 find_package(PkgConfig REQUIRED)
 pkg_check_modules(CCD REQUIRED ccd)
 
-find_package(gazebo 7 REQUIRED)
+find_package(gazebo 7 QUIET)
+if (gazebo_FOUND)
+  message(STATUS "gazebo7 found!")
+else()
+  message(STATUS "gazebo7 not found, searching for gazebo8")
+  find_package(gazebo 8 QUIET)
+  if (gazebo_FOUND)
+    message(STATUS "gazebo8 found!")
+  else()
+    message(STATUS "gazebo8 not found, searching for gazebo9")
+    find_package(gazebo 9 QUIET)
+    if (gazebo_FOUND)
+      message(STATUS "gazebo9 found!")
+    else()
+      message(STATUS "gazebo9 not found, searching for gazebo10")
+      find_package(gazebo 10 QUIET)
+    endif()
+  endif()
+endif()
+if (NOT gazebo_FOUND)
+  message(FATAL_ERROR "Could not find a version of gazebo >= 7")
```

Comment:
It's awkwards to have to list all the individual gazebo versions in a cascade search that will need to be extended for each new version. Is there a generic one that will find the latest and then a version compare can be used?

Reply:
Yeah, it is awkward. `pkg-config` has the `>=` syntax, which would be nice. The edge case is if you have two versions of gazebo installed, one of which is too old, it might only find the old one. That is a weird edge case though.